src/api/docker-compose.yaml

```diff
@@ -16,14 +16,67 @@ services:
       - '8080:8080'
     volumes:
       - /var/run/docker.sock:/var/run/docker.sock
+      - ./traefik/config/traefik.yaml:/etc/traefik/traefik.yaml"
```

Comment:
Stray `"` at the end?

Also, are we using this file (we don't have one in the tree)?  Should we add one or do it all through labels?